### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by ScreamingHawk** [ScreamingHawk/android-slideshow#26]
Slideshow is at a stable point in development and I don't expect to see many/any changes to the strings.xml for a while. 

Would be nice is some bilingual people could translate the application into various languages. 

Any languages is appreciated, but I have installs in the following non-English countries: 
* Russia
* Belgium
* India
* Austria
* China
* Germany
* Italy
* Netherlands
* Denmark
* Poland

**Comment by laura-barluzzi** [comment]
Hi @ScreamingHawk, I'm Italian and I'm interested in translating to Italian whatever you need. Could you please guide me to the right place to add my translations?

**Comment by ScreamingHawk** [comment]
@laura-barluzzi Thanks Laura! Have a look at the contributing information I just put up [here](https://github.com/ScreamingHawk/android-slideshow/blob/master/CONTRIBUTING.md) and let me know if you still have questions

**Comment by ScreamingHawk** [comment]
You can have a look at the Lithuanian translations in `values-lt` as an example :)

**Comment by thlabbe** [comment]
thanks to https://open-source.now.sh/ and https://news.ycombinator.com/item?id=13908606

**Comment by ScreamingHawk** [comment]
@thlabbe Exactly why I added these labels!! I still find it really hard to find projects I can contribute to in meaningful but easy ways. This website is a fantastic idea, and I'm happy for all the help.

**Comment by JekRock** [comment]
Hi! I added a Russian translation to your app. #34

**Comment by TheRojam** [comment]
i would do **German** Translations

**Comment by ScreamingHawk** [comment]
@TheRojam We already have a German translation. You can find it [here](https://github.com/ScreamingHawk/android-slideshow/blob/master/app/src/main/res/values-de/strings.xml). If you feel there is a mistake (I don't speak German so I wouldn't know..) you can fork and create a pull request for fixes, otherwise this has already been done.

**Comment by TheRojam** [comment]
@ScreamingHawk bad german --  looks like google translate... bad grammar. where i can finde the original?

**Comment by ScreamingHawk** [comment]
@TheRojam Have a look at the [contributing](https://github.com/ScreamingHawk/android-slideshow/blob/master/CONTRIBUTING.md) page if you want to fix it. Instead of creating a new file, just overwrite the existing German one. 

Unfortunately I have no way of knowing if a translation is good or not so I have to put my faith in the contributors.

**Comment by ScreamingHawk** [comment]
@mobtechpd
@laura-barluzzi
@SentToDevNull
@thlabbe
@aym3nb
@zephyrict
@uzluisf
@KristianTurner

I'm adding another feature (#36) that has 3 additional strings values. If you could update the translations to include these new values it would be much appreciated :tada: 

I'm planning to release the feature to stable in the next week or so. 

(Please don't feel obligated to do these translations. It's your choice and I appreciate all the help I've received so far)

**Comment by lbaca** [comment]
Hi!

@uzluisf did a great job with the Spanish translations, but I was wondering if it would be OK for me to make a couple of improvements, and take this as an opportunity to submit my first-ever pull request!

Any objections?

**Comment by ScreamingHawk** [comment]
@lbaca No objections. Please feel free. 

As I don't speak languages other than English I can never be sure if the translations are any good, so I'm always grateful when improvements are offered.

**Comment by ScreamingHawk** [comment]
This project has been sitting around for a while, but I've made a couple updates and seen more installs than I was expecting. 

A great deal of this has been thanks to all the help from the translators! Thanks so much. You guys deserve more credit than you've been given so I've added another line in the credits just for you!! 

Please update the translation for your language to include the new credits. 
```
	<string name="credits_content6">This translation of &appname; is brought to you by
		<a href="https://michael.standen.link">&author;</a>.
	</string>
```
Stick your name (or pseudonym) in place of `&author;` and change the `href` to be a link to your site or Github profile or `mailto:your@email.com`. 

Lithuanian: @mobtechpd
Italian: @laura-barluzzi
German: @SentToDevNull
French: Thomas Labbe
Arabic: @aym3nb
Dutch: @zephyrict, Dave
Spanish: @uzluisf, @lbaca 
Danish: @KristianTurner 
Russian: @JekRock, @BalabanovGit 
Japanese: @naofum 
Swedish: @cadii 

For the translation with multiple contributors, can you please include both translators. Thanks

**Comment by lbaca** [comment]
I'll handle the Spanish.

**Comment by aym3nb** [comment]
Thanks @ScreamingHawk

**Comment by ASVS-Kartheek** [comment]
Hi I'm Indian. There are many major languages in India, I would like to translate this into Telugu if it is helpful.

**Comment by ScreamingHawk** [comment]
@ASVS-Kartheek Yes please!

**Comment by sumanthrao** [comment]
I would like to add Indian language translations(Hindi and Kannada).Could you guide me to the concerned files?

**Comment by ScreamingHawk** [comment]
@sumanthrao Yes please! Directions are at the [contributing](https://github.com/ScreamingHawk/android-slideshow/blob/master/CONTRIBUTING.md) page

**Comment by MelkyFB** [comment]
I like to add Portuguese language translation, please refer to my push: https://github.com/MelkyFB/android-slideshow/commit/b981e75d4f814b388c95daddbbef40ec7d1996f1

**Comment by ScreamingHawk** [comment]
Hi all, 

I've added another setting that can be translated at your leisure. 
Thanks again for any help you provide with this. 

Cheers,
Michael

**Comment by wardbradt** [comment]
Yes please 


Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: Liuxi-G <notifications@github.com> Date: 11/21/17  2:53 PM  (GMT-05:00) To: ScreamingHawk/android-slideshow <android-slideshow@noreply.github.com> Cc: Ward Bradt <wbradt@optonline.net>, Mention <mention@noreply.github.com> Subject: Re: [ScreamingHawk/android-slideshow] Translations (#26) 
Hello. I found something to improve in the translation of Chinese Version,can I claim this issue and  take the task of modifing the translation?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/ScreamingHawk/android-slideshow","title":"ScreamingHawk/android-slideshow","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/ScreamingHawk/android-slideshow"}},"updates":{"snippets":[{"icon":"PERSON","message":"@Liuxi-G in #26: Hello. I found something to improve in the translation of Chinese Version,can I claim this issue and  take the task of modifing the translation?"}],"action":{"name":"View Issue","url":"https://github.com/ScreamingHawk/android-slideshow/issues/26#issuecomment-346141965"}}}

**Comment by JekRock** [comment]
@ScreamingHawk I can add a Ukrainian translation if you're interested in it

**Comment by ScreamingHawk** [comment]
I'm always happy to accept pull requests @JekRock :) Feel free
